plugins/source/aws/Makefile

```diff
@@ -18,3 +18,8 @@ install-tools:
 .PHONY: install-hooks
 install-hooks:
 	pre-commit install
+
+.PHONY: gen-docs
+gen-docs:
+	go run main.go doc ./docs/tables
```

Comment:
Maybe delete `docs/tables` here too (that's how our CI does it) to ensure removal of renamed tables